src/Workleap.DomainEventPropagation.Tests/Publishing/EventPropagationPublisherOptionsValidatorTests.cs

```diff
@@ -14,6 +14,13 @@ public class EventPropagationPublisherOptionsValidatorTests
     [InlineData("topicName", " ", "http://topicurl.com", false, false)]
     [InlineData("topicName", "accessKey", "http://topicurl.com", true, true)]
     [InlineData("topicName", "accessKey", "http://topicurl.com", false, true)]
+    [InlineData(null, null, null, false, false)]
+    [InlineData(" ", "accessKey", "http://topicurl.com", false, false)]
+    [InlineData(null, "accessKey", "http://topicurl.com", false, false)]
+    [InlineData("topicName", null, "http://topicurl.com", false, false)]
+    [InlineData("topicName", "accessKey", " ", false, false)]
+    [InlineData("topicName", "accessKey", null, false, false)]
+    [InlineData("topicName", "accessKey", "topicEndpoint", false, false)]
     public void GivenEventPropagationConfigurations_WhenValidateOptions_ThenAccessCredentialsValidated(string topicName, string topicAccessKey, string topicEndpoint, bool useTokenCredential, bool optionsValid)
```

Comment:
I think we can remove the test named GivenEventPropagationConfiguration_WhenOptionsAreInvalid_ThrowsException in the EventPropagationPublisherOptionsTests class which validated the attributes.

Reply:
I wasn't sure, but you are right